src/managed/OpenLiveWriter.BlogClient/Clients/GoogleBloggerv3Client.cs

```diff
@@ -88,7 +88,7 @@ private static BlogPost ConvertToBlogPost(Post post)
                 Permalink = post.Url,
                 Contents = post.Content,
                 DatePublished = post.Published.Value,
-                Keywords = string.Join(new string(LabelDelimiter,1), post.Labels ?? new List<string>())
+                Categories = post.Labels.Select(x => new BlogPostCategory(x)).ToArray()
```

Comment:
This needs to be `post.Labels?.Select(x => new BlogPostCategory(x)).ToArray()`. post.Labels can be null, so I'm getting a NullReferenceException on some posts when I go to **File** > **Open recent post** > **My Test Blog**.